File: node/examples/phoneset.ts
import 'dotenv/config'

import { TtsClient, speechAudioFormat_CodecToJSON, speechAudioFormat_ContainerToJSON, voice_GenderToJSON } from '@aristech-org/tts-client'

const auth = process.env.TOKEN && process.env.SECRET ? { token: process.env.TOKEN, secret: process.env.SECRET } : undefined

const client = new TtsClient({
  host: process.env.HOST,
  ssl: Boolean(auth) || Boolean(process.env.ROOT_CERT) || process.env.SSL === 'true',
  rootCert: process.env.ROOT_CERT,
  auth,
})
const { phoneset } = await client.getPhoneset({ voice: { voiceId: process.env.VOICE_ID || 'anne_en_GB' } })
console.log(JSON.parse(phoneset))

File: python/examples/phoneset.py
import json

from aristech_tts_client import TtsClient, PhonesetRequest, Voice
from utils import host, auth_token, auth_secret, root_cert, ssl, voice_id

client = TtsClient(host=host, ssl=ssl, root_cert=root_cert, auth_token=auth_token, auth_secret=auth_secret)
response = client.get_phoneset(PhonesetRequest(voice=Voice(voice_id=voice_id)))
print(response.phoneset)